docker/docker-compose.yml

```
@@ -1,61 +1,74 @@
 version: '3.7'
+x-default-redis: &default-redis
+  image: redis:${REDIS_VERSION:-6.2.5-alpine}
+  deploy:
+    restart_policy:
+      condition: any
+  volumes:
+    - ${APP_HOME:-..}/redis:/data
+  command: redis-server --appendonly yes
+
+x-environment: &environment
+  PASSENGER_APP_ENV: test
+  REDIS_COUNT_HOST: redis-count
+  REDIS_HOST: redis-cache
+  REDIS_RESQUE_HOST: redis
+
+x-default: &default-api
```

Can we have a uniform naming convention?
```suggestion
x-redis: &default-redis
  image: redis:${REDIS_VERSION:-6.2.5-alpine}
  deploy:
    restart_policy:
      condition: any
  volumes:
    - ${APP_HOME:-..}/redis:/data
  command: redis-server --appendonly yes

x-environment: &default-environment
  PASSENGER_APP_ENV: test
  REDIS_COUNT_HOST: redis-count
  REDIS_HOST: redis-cache
  REDIS_RESQUE_HOST: redis

x-api: &default-api
```